odata-protocol/11.4 Data Modification.md

```diff
@@ -178,13 +178,22 @@ values beyond those specified in the metadata SHOULD NOT be sent in the
 request body. The service MUST fail if unable to persist all property
 values specified in the request.
 
-Properties computed by the service (annotated with the term
-[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed),
-see [OData-VocCore](#ODataVocCore)) and properties that are tied to
-properties of the principal entity by a referential constraint, can be
-omitted and MUST be ignored if included in the request.
-
-Properties with a defined default value, nullable properties, and
+Non-insertable properties, as well as dependent properties that are tied to
+non-key properties of the principal entity, SHOULD be omitted from the request body.
+If provided, services MUST either ignore the value in the request body or fail the request.
+
+Non-insertable properties include (and are not limited to)
+- properties annotated with the term
+[`Core.Computed`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Computed), see [OData-VocCore](#ODataVocCore),
+- properties listed as `NonInsertableProperties` of term [`Capabilities.InsertRestrictions`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Capabilities.V1.md#InsertRestrictions), see [OData-VocCap](#ODataVocCap),
+- properties annotated with term
+[`Core.Permissions`](https://github.com/oasis-tcs/odata-vocabularies/blob/main/vocabularies/Org.OData.Core.V1.md#Permissions), see [OData-VocCore](#ODataVocCore), where the annotation value does not have the `Write` flag.
+
+Services MUST return an error if the request body contains a value for a
+property marked as insertable that cannot currently be inserted by the
```

Comment:
marked as insertable = listed as `NonInsertableProperties`?

Reply:
Then I suggest
```suggestion
such a property that cannot currently be inserted by the
```
"marked as insertable" sounds like a new (undefined) term

Reply:
Actually it would be "any property except such a property"

Reply:
You are right, I updated my suggestion.